### PR TITLE
Sass instead of Less

### PR DESCRIPTION
**Pull request by msx2**
@arendjr Here's the first part of what I'm planning, like I mentioned on Twitter :smiling_imp: 
If you have any remarks or would like something to be done with a different/better approach, let me know :smile:

### CODE REVIEW

**Comment by arendjr** [comment]
Looks like a great PR!

I have a few minor points, though I admit I need to look a little bit more in depth when I have a bit more time:

- Looks like the file selectivity.sass is generated and should not be checked in? If so, I think it makes sense to add to .gitignore

- The Sass files use two spaces for indentation, where I prefer four spaces.

- Is it possible to maintain comments in the output of the non-minified dist file? I think some users would prefer that.

**Comment by msx2** [comment]
As for `selectivity.sass` - well, I've originally used it in `sass` task to perform a build, but then again looked through the build and restored custom build. Right now this file does not serve any special purpose so I guess it would be possible to remove it completely and add to `.gitignore` if you prefer it this way.

As for spaces - yeah, sorry for that. I'm used to ruby convention of two spaces. Will fix that!

As for comments - sure, I will restore them.

**Comment by msx2** [comment]
@arendjr Changes done :)

File: gulp/tasks/sass.js [hunk 1]
```diff
@@ -15,19 +16,23 @@ module.exports = function() {
...
     CustomBuildUtil.checkOptions(argv);
 
-    fs.writeFileSync('styles/selectivity-custom.less', argv.modules.map(function(module) {
-        if (fs.existsSync('styles/selectivity-' + module + '.less')) {
-            return '@import "selectivity-' + module + '";';
-        } else {
-            return '';
-        }
+    // Config files!
+    argv.sassModules = argv.modules.slice();
```
Comment (arendjr) [inline gulp/tasks/sass.js]: No need to assign this to the argv object. `var sassModules = ...` would do the trick, I think.

Comment (msx2) [inline gulp/tasks/sass.js]: Yeah, pretty much it did. Updated :)

**Comment by arendjr** [comment]
Wow, that was quick! Looks great, I'm gonna merge!

**Pull request merged**